### PR TITLE
[NFC] This is only a temp change to test email notifications on PR.

### PR DESCRIPTION
**Pull request by zahiraam**

### CODE REVIEW

**Comment by smanna12** [comment]
LGTM

File: clang/test/CodeGen/dummy-test.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 -emit-llvm %s -o /dev/null
```
Comment (smanna12) [inline clang/test/CodeGen/dummy-test.cpp:1]: Missing comment

**Comment by zahiraam** [comment]
@smanna12 Would you mind commenting again. I am trying to cleanup my junk email folder and want to make sure that I receive email from your comment from this PR. Thanks.

**Comment by smanna12** [comment]
Need to include mock header in your test

**Pull request closed without merging**